install/resources/mybb_theme.xml

```diff
@@ -448,6 +448,14 @@ table {
 	color: #999;
 }
 
+.lastpost_subject {
+	display: inline-block;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	width: 200px;
```

Comment:
Can't the width be `100%` or removed? I know the default theme isn't mobile friendly but we won't want to make it worse.

Reply:
I dont think so... now it uses same width as td in table.